### PR TITLE
Connect price filter

### LINKED ISSUE #129: Hotels: connect price filter to the API
Related issue: #97

### PR DESCRIPTION
**Pull request by jakubzaba**
Closes #129

---
Please check this before merging (do not delete this checklist):

- [ ] it works in landscape and portrait mode
- [ ] it uses `cacheConfig.force=true` if offline access doesn't make sense

### CODE REVIEW

File: app/hotels/src/filter/FilterStripe.js [hunk 1]
```diff
@@ -17,26 +18,55 @@ const styles = {
...
+    minPrice: number,
+    maxPrice: number,
+  }) => {
+    const filter = (({}: any): FilterParametersType);
+    if (minPrice !== MIN_PRICE) {
+      filter.minPrice = minPrice;
+    }
+    if (maxPrice !== MAX_PRICE) {
+      filter.maxPrice = maxPrice;
+    }
```
Comment (mrtnzlml) [inline app/hotels/src/filter/FilterStripe.js]: I don't think all this is a responsibility of `FilterStripe`. It's just a stripe so it should care only about being stripe (not about prices and stars and user rating and ... and ...). 😊

File: app/hotels/src/__tests__/HotelsReducer.test.js [hunk 2]
```diff
@@ -22,9 +22,11 @@ describe('HotelsReducer', () => {
       const action = {
         type: 'setSearchFilters',
         filter: {
-          ...defaultSearchParams,
-          latitude: 50.5,
-          checkin: new Date(1),
+          search: {
```
Comment (jaroslav-kubicek) [inline app/hotels/src/__tests__/HotelsReducer.test.js]: I would personally advise having separated action for filters

File: app/hotels/src/filter/FilterStripe.js [hunk 3]
```diff
@@ -17,26 +18,36 @@ const styles = {
...
+  handleStarsChange = () => {
+    // TODO stars filter handler
   };
 
+  handlePriceChange = (filter: FilterParametersType) =>
+    this.props.onChange(filter);
+
   render = () => {
+    const { filter } = this.props;
+    const minPrice = (filter && filter.minPrice) || MIN_PRICE;
```
Comment (jaroslav-kubicek) [inline app/hotels/src/filter/FilterStripe.js]: From my perspective, this should be still moved to PriceFilter itself and you don't need to import max/min price as well.

**Comment by jakubzaba** [comment]
I made few more changes, take a look on whole PR again, pls.

**Pull request merged**